recipes/language-detection

```diff
@@ -0,0 +1,3 @@
+(language-detection :fetcher github
+                    :repo "andreasjansson/language-detection.el"
+                    :files ("language-detection.el"))
```

Comment:
You need not to specify `:files` attribute, it is enough to use default value.

Please read 
- https://github.com/melpa/melpa#recipe-format